space-age/SpaceAgeTest.m

```diff
@@ -0,0 +1,62 @@
+#import <XCTest/XCTest.h>
+#import "SpaceAge.h"
+
+@interface test_suite : XCTestCase
+
+@end
+
+@implementation test_suite
+
+- (void)testAgeInSeconds {
+    SpaceAge *age = [[SpaceAge alloc] initWithSeconds:1000000];
+    XCTAssertEqual(1000000, [age seconds]);
+}
+
+- (void)testAgeInEarthYears {
+    SpaceAge *age = [[SpaceAge alloc] initWithSeconds:1000000000];
+    XCTAssertEqual(31.69, [age onEarth]);
```

Comment:
How about using `XCTAssertEqualWithAccuracy` and a constant accuracy of 0.01 or 0.005 for this test and all of the following ones?  I can see having an exact equality check being frustrating for people.